frontend/src/scene/main-view/CourseModal.js

```diff
@@ -82,15 +108,29 @@ const PaymentSection = styled(Content)`
         }
     }
 `;
+const ReservationInfoArea = posed.div({
+    show: {
+        y: 0,
+    },
+    normal: {
+        y: 0,
+    },
+});
 
 class CourseModal extends React.Component {
     clear = (e) => {
         this.props.courseStore.selectCourse(null);
+        this.props.courseStore.clearConfirmationModal();
     };
     render() {
         const course = this.props.courseStore.courseInFocus;
+        const reservedCourse = this.props.courseStore.reservedCourse;
```

Comment:
What is this for?

Reply:
this is for getting the reserved course.